src/_assets/css/_bootstrap_config.scss

```diff
@@ -17,7 +17,13 @@ $h2-font-size: 2.25rem;
 $h3-font-size: 1.125rem;
 $dt-font-weight: $font-weight-base;
 
-// Spacing (extra spacers beyond Bootstrap's defaults)
+// Layout config
+$container-max-widths: (
+  sm: 540px,
+  md: 720px,
+  lg: 960px,
+  xl: 1330px
```

Comment:
Why deviate from the default xl 1140px?
Maybe define xxl instead as the key for 1130px?